### ISSUE TITLE
Block mover not actionable with wide and full alignments

### ISSUE THREAD

**Issue by afercia** [WordPress/twentynineteen#665]
I've read the conversation on https://github.com/WordPress/twentynineteen/pull/614 which recently made a few changes to the block toolbar position. However, I'm not fully convinced a theme should ever try to modify the Gutenberg UI, as this leads to potential breakage.

With those changes, the block mover is not usable when the alignment is set to "wide" of "full". 

Normally, the block mover is placed vertically on the left of the block:

![screenshot 2018-11-25 at 13 36 35](https://user-images.githubusercontent.com/1682452/48980805-102b4300-f0ce-11e8-8d34-a2a6335b2043.jpg)

With a "wide" alignment, it's expected to be placed horizontally and now it's almost completely hidden behind the toolbar:

![screenshot 2018-11-25 at 13 36 49](https://user-images.githubusercontent.com/1682452/48980812-233e1300-f0ce-11e8-809c-21f8a7b95e5b.jpg)

With a "full" alignment, it's completely hidden:

![screenshot 2018-11-25 at 13 36 56](https://user-images.githubusercontent.com/1682452/48980816-32bd5c00-f0ce-11e8-859c-c23e254754d9.jpg)

It can be used only when the block is _not_ selected and the toolbar doesn't appear, i.e. when the block is in the "hovered" state, which is less than ideal:

![screenshot 2018-11-25 at 16 22 48](https://user-images.githubusercontent.com/1682452/48980847-7f089c00-f0ce-11e8-9ac5-fd241cfd6eed.png)

As mentioned in https://github.com/WordPress/twentynineteen/pull/614, maybe Gutenberg itself should better handle this cases. 

For example, if Twenty Nineteen reverts the change to the toolbar position, that would help for the "Full" alignment:

![screenshot 2018-11-25 at 14 59 37](https://user-images.githubusercontent.com/1682452/48980947-ccd1d400-f0cf-11e8-9cab-5414c8f810fa.jpg)

But it wouldn't help for the "wide" alignment because in Twenty Nineteen "wide" is left aligned and there's really no space on the left to display the block movers in their expected horizontal position:

<img width="1006" alt="screenshot 2018-11-25 at 16 39 53" src="https://user-images.githubusercontent.com/1682452/48981040-05be7880-f0d1-11e8-82b4-def65613665b.png">

Basically this has also to do with the concept of "content width" and the way themes are supposed to alter the `wp-block` width in the editor (as Twenty Nineteen does).

I'd tend to think this second part should be addressed in Gutenberg, going to create a new issue there.

/Cc @jasmussen

**Comment by jasmussen** [comment]
I'll take a look at this one.
